ClassAttendance/src/ConsoleUI/Views/Implementations/SubMenus/LecturerMenuView.cs

```diff
@@ -0,0 +1,79 @@
+﻿using BLL.Models;
+using ConsoleUI.Models.Lecturer;
+using ConsoleUI.Views.Interfaces;
+using System;
+using System.Collections.Generic;
+
+namespace ConsoleUI.Views.Implementations.SubMenus
+{
+    public class LecturerMenuView : IMenuView
```

Comment:
First: it's like an IConsoleHelper, not an IMenuView because of the functionality here more than just view.
Second: in this case, it would be better to have more common methods signatures (like Print) in IConsoleHelper and create a separate interface for each Entity (ILecturerConsoleHelper) that will be inherited from IConsoleHelper.
Third: I think the PrintMenu method should be a part of the Controller because the switch-case allocated in the Controller.

Reply:
1) I'm trying to use MVC pattern here. This classed are something like MVC's views. All inputs are handled in views and only passes data to controller.
2) I will fix to Controller classes to call View Print method.
3) See 1)

Reply:
Okay.